From 9f938a8168298963a38d58b6eaaebf6e0de56cc3 Mon Sep 17 00:00:00 2001
From: Avik Sil <avik.sil@linaro.org>
Date: Thu, 16 Feb 2012 19:55:40 +0530
Subject: [PATCH] Define MESA_EGL_NO_X11_HEADERS for building with mesa
 headers

This definition avoids including <X11/Xlib.h> that prevents build
errors with mesa egl headers while building xbmc/interfaces/json-rpc/JSONServiceDescription.cpp
---
 xbmc/interfaces/json-rpc/Makefile |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/xbmc/interfaces/json-rpc/Makefile b/xbmc/interfaces/json-rpc/Makefile
index 648ec24..078b6a8 100644
--- a/xbmc/interfaces/json-rpc/Makefile
+++ b/xbmc/interfaces/json-rpc/Makefile
@@ -1,3 +1,5 @@
+CXXFLAGS+=-DMESA_EGL_NO_X11_HEADERS
+
 SRCS=ApplicationOperations.cpp \
      AudioLibrary.cpp \
      FileItemHandler.cpp \
-- 
1.7.5.4

